### PR TITLE
[#167392841] Retrieve municipalities metadata

### PR DESCRIPTION
**Pull request by Undermaken**
This PR aims to introduce all logics needed to retrieve municipalities metadata

Municipality metadata is a static content provided by [io-services-metadata](https://github.com/teamdigitale/io-services-metadata) 
To know more about how municipality metadata is created and where it is placed [PR#53](https://github.com/teamdigitale/io-services-metadata/pull/53)

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by Undermaken** [comment]
@fpersico could you review this PR? It is a blocker for another working [PR](https://github.com/teamdigitale/io-app/pull/1206) 
Thanks ;)

File: ts/store/actions/content.ts [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
 export const contentServiceLoad = createAsyncAction(
   "CONTENT_SERVICE_LOAD_REQUEST",
   "CONTENT_SERVICE_LOAD_SUCCESS",
   "CONTENT_SERVICE_LOAD_FAILURE"
 )<ServiceId, { serviceId: ServiceId; data: ServiceMetadata }, ServiceId>();
 
-export type ContentActions = ActionType<typeof contentServiceLoad>;
+export const contentMunicipalityLoad = createAsyncAction(
+  "CONTENT_MUNICIPALITY_LOAD_REQUEST",
+  "CONTENT_MUNICIPALITY_SUCCESS",
```
Comment (fpersico) [inline ts/store/actions/content.ts]:
```suggestion
  "CONTENT_MUNICIPALITY_LOAD_SUCCESS",
```

File: ts/store/actions/content.ts [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+  "CONTENT_MUNICIPALITY_FAILURE"
```
Comment (fpersico) [inline ts/store/actions/content.ts]:
```suggestion
  "CONTENT_MUNICIPALITY_LOAD_FAILURE"
```

File: ts/sagas/contentLoaders.ts [hunk 3]
```diff
@@ -52,3 +58,49 @@ export function* watchContentServiceLoadSaga(): Iterator<Effect> {
     }
   });
 }
+
+/**
+ * Retrieves a municipality metadata from the static content repository
+ */
+function getMunicipalityMetadata(
```
Comment (fpersico) [inline ts/sagas/contentLoaders.ts]: This can be a saga.

Comment (Undermaken) [reply ts/sagas/contentLoaders.ts]: @fpersico I created `watchContentMunicipalityLoadSaga` generator. See line [85](https://github.com/teamdigitale/io-app/blob/55191eaac468736a948712581dedd7e53624aa75/ts/sagas/contentLoaders.ts#L85)

Comment (Undermaken) [reply ts/sagas/contentLoaders.ts]: done in commit https://github.com/teamdigitale/io-app/pull/1220/commits/acd5adce7854aeeef69ed82e9e0b8570f507bdd8

**Pull request merged**